tests/sles4sap/qesapdeployment/test_cluster.pm

```diff
@@ -24,13 +24,25 @@ sub run {
     record_info("crm status", $cmr_status);
     qesap_ansible_cmd(cmd => $crm_mon_cmd, provider => $prov, filter => '"hana[0]"');
     qesap_cluster_logs();
+
+    if (get_var("QESAP_IBSMIRROR_RESOURCE_GROUP")) {
+        my $rg = qesap_get_az_resource_group();
+        my $ibs_mirror_rg = get_var('QESAP_IBSMIRROR_RESOURCE_GROUP');
+        qesap_az_vnet_peering(source_group => $rg, target_group => $ibs_mirror_rg);
+        qesap_delete_az_peering(source_group => $rg, target_group => $ibs_mirror_rg);
```

Comment:
Is this OK? Delete peering right after setting it up?

Reply:
it is intentionally the bare minimal test to validate the new API

Reply:
I guess this is a test setting, i.e. tries to test the peering without actually applying any updates.

However, we probably shouldn't merge it if it's only for testing? As far as I understand, the peering will happen either in a separate module or during the module that adds the repos, probably not in `test_cluster` (but Michele may have another idea)

Reply:
Why not to merge this even if the peering is only to test the API? As we are using the WORKER_ID as seed we will not "steal" any resources (like IP ranges).

Reply:
Why not to merge this even if the peering is only to test the API? As we are using the WORKER_ID as seed we will not "steal" any resources (like IP ranges). The purpose of this is really only to probe the peering create and destroy AZ commands, the `vnet_address_range` and `subnet_address_range` settings in qe-sap-deployment and eventually partially the idea about using the WORKER_ID as seed

Reply:
True, one would say that this may even be a part of testing the cluster, so it belongs here anyway. Fine to me